nixos/modules/virtualisation/virtualbox-guest.nix

```diff
@@ -52,7 +54,7 @@ in
       description = "Whether to enable seamless mode. When activated windows from the guest appear next to the windows of the host.";
     };
 
-    draganddrop = mkOption {
+    dragAndDrop = mkOption {
```

Comment:
This change of module parameters should not be backported. We typically don't do this on release branches, do we?

Reply:
Yeah, I added it due to [this](https://github.com/NixOS/nixpkgs/pull/316400#discussion_r1627348536) comment, tbh.

I'll undo the rename-related changes